### PR TITLE
RFC: Jira-CDEvents Integration

### PR DESCRIPTION
**Pull request by rjalander**
This RFC describes 
- an approach to integrate Jira with CDEvents using using [webhook-adapter](https://github.com/cdevents/webhook-adapter).
- and the list of Jira event types mapping with CDEvents

### CODE REVIEW

File: rfc/jira-translator-cdevents.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
...eCount\":0,\"state\":\"OPEN\",\"details\":{\"openCount\":0,\"mergedCount\":0,\"declinedCount\":0,\"total\":0},\"open\":true},\"byInstanceType\":{}},\"build\":{\"overall\":{\"count\":0,\"lastUpdated\":null,\"failedBuildCount\":0,\"successfulBuildCount\":0,\"unknownBuildCount\":0},\"byInstanceType\":{}},\"review\":{\"overall\":{\"count\":0,\"lastUpdated\":null,\"stateCount\":0,\"state\":null,\"dueDate\":null,\"overDue\":false,\"completed\":false},\"byInstanceType\":{}},\"deployment-environment\":{\"overall\":{\"count\":0,\"lastUpdated\":null,\"topEnvironments\":[],\"showProjects\":false,\"successfulCount\":0},\"byInstanceType\":{}},\"repository\":{\"overall\":{\"count\":0,\"lastUpdated\":null},\"byInstanceType\":{}},\"branch\":{\"overall\":{\"count\":0,\"lastUpdated\":null},\"byInstanceType\":{}}}},\"isStale\":false}}","aggregateprogress":{"progress":0,"total":0},"environment":null,"duedate":null,"progress":{"progress":0,"total":0},"comment":{"comments":[],"maxResults":0,"total":0,"startAt":0},"votes":{"self":"http://localhost:9090/rest/api/2/issue/ES-24/votes","votes":0,"hasVoted":false},"worklog":{"startAt":0,"maxResults":20,"total":0,"worklogs":[]},"archivedby":null}}}</details> |
```
Comment (xibz) [inline rfc/jira-translator-cdevents.md:22]: Can we prettify these JSON examples and put them in `<pre>`

Comment (rjalander) [reply rfc/jira-translator-cdevents.md]: formatted event json using links, outside the table

**Comment by rjalander** [comment]
@xibz could please approve if there are no other comments and merge.
Also please create a repository `jira-translator` under `cdevents` once this is merged.
Thank you.

**Review by xibz — APPROVED** [review]
Awesome! This looks good! Thanks for the updates @rjalander

**Comment by rjalander** [comment]
@xibz @afrittoli, could you please a create repository `jira-translator`

**Review by afrittoli — APPROVED** [review]

**Pull request merged**